lib/common/api/crash-reporter.js

```diff
@@ -43,7 +43,12 @@ class CrashReporter {
       throw new Error('companyName is a required option to crashReporter.start')
     }
     if (submitURL == null) {
-      throw new Error('submitURL is a required option to crashReporter.start')
+      if (uploadToServer) {
+        throw new Error('submitURL is a required option to crashReporter.start')
+      } else {
+        // use a dummy upload URL
+        submitURL = 'http://dummy/upload/url'
```

Comment:
> later call setUploadToServer(true) and then get a crash, you'll see your machine make a request to this URL right

Yes, I considered using `http://localhost` but it is possible that the user has a local http server.